drivers/base/bootvid/arm/bootvid.c

```diff
@@ -103,9 +104,10 @@ VgaScroll(IN ULONG Scroll)
 
 VOID
 NTAPI
-PreserveRow(IN ULONG CurrentTop,
-            IN ULONG TopDelta,
-            IN BOOLEAN Restore)
+VidpPreserveRow(
```

Comment:
```suggestion
PreserveRow(
```
to emphasise it's a per-arch dependent global helper function and not a generic private bootvid helper.